### DISCUSSION TITLE
Why were the patch versions for CVE-2021-28955 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [git-bug/git-bug#1137] [Q&A]
We are a research team dedicated to Golang, have discovered that CVE-2021-28955 was addressed in commit 01b949091290a3bfd5870c6457c941db1abf3d67. However, upon analyzing the commit, we observed that the patch version (v0.8.0) was released after a lapse of over one month. We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users. We seek clarification on whether the delay might be attributed to:

1. Issues with testing and CI checking.
2. Other commits requiring inclusion in a single release.
3. By convention, infrequent release of versions.
4. Other reasons.
   We appreciate your attention to this matter and eagerly await your response. Thank you.

**Comment by 20after4** [comment]
The README for this project mentions that the maintainer is finding it to be a lot of work to maintain the project and asking for help with project maintenance.  I would guess that is the reason.

**Comment by MichaelMure** [comment]
I was informed of that issue March 20, 2021 by email. I released a fixed version (https://github.com/MichaelMure/git-bug/releases/tag/v0.7.2) and filled a security advisory on Github the next day.

So the answer is is "4. It wasn't late".
